example/src/main/java/io/netty/example/http/websocketx/benchmarkserver/WebSocketServerHandler.java

```diff
@@ -47,8 +50,13 @@
 
     private static final String WEBSOCKET_PATH = "/websocket";
 
+    private final Set<ChannelHandler> httpHandlers;
     private WebSocketServerHandshaker handshaker;
 
+    public WebSocketServerHandler(Set<ChannelHandler> httpHandlers) {
```

Comment:
- Is this constructor supposed to be public? If not, let's make it package-local
- This effectively removes the default constructor. Is this intended?